app/classifier/tasks/generic-editor.cjsx

```diff
@@ -57,6 +57,14 @@ module.exports = React.createClass
         </div>}
       {' '}
 
+      {if 'hide previous marks' in @props.project.experimental_tools
```

Comment:
I truly do not believe this should be optional. But if we're going to make it optional, and we persist the checkbox across tasks, this should be a property of the workflow. If it's only on the drawing task, this is fine.